addons/binding/org.openhab.binding.smappee/README.md

```diff
@@ -0,0 +1,132 @@
+# Smappee Binding
+
+This binding integrates the [Smappee energy monitor](http://www.smappee.com/be_en/home).
+
+## Introduction
+
+Thanks to the Smappee energy monitor you always know how much power your appliances consume, wherever you are. For more comfort, a better insight and sustainable energy savings up to 30%.
+
+Both the Energy monitor and the Solar energy monitor can be used. These can be bought online in the [smappee webshop] (http://www.smappee.com/be_en/eshop/monitors.html).
+
+The smappee supports clamps for 3 phase and 1 phase systems and there is a separate circuit clamp for your solar system. installation manuals can be found on [their support page](https://www.smappee.com/be_en/support)
+
+## Supported Things
+
+This binding provides 1 bridge thing type : 'smappee'. There should be only 1 smappee device installed in your home.
+This thing act like a bridge and represents your smappee monitor device and should be configured first. Once configured, this will autodetect all other smappee things.
+Detected things :
+
+- smappee-appliance: A detected appliance, the appliances that are not categorized yet (type "Find Me"), are skipped. 
+- smappee-actuator: A paired smappee switch.
+- smappee-sensor: A paired water or gas sensor.
+
+## Discovery
+
+The smappee thing must be configured manually. Appliances, Actuators and the sensors are auto-detected. 
```

Comment:
changed to "autodetected" to make it consistent, this is also consistent through the whole Smappee binding where "autodetect" is used instead of "auto-detect"